ui/src/components/friends/add.rs

```diff
@@ -4,12 +4,17 @@ use std::str::FromStr;
 use common::language::get_local_text;
 use dioxus::prelude::*;
 use futures::{channel::oneshot, StreamExt};
+use kit::components::context_menu::{ContextItem, ContextMenu};
+use kit::elements::Appearance;
```

Comment:
Only if its not going to be used since clippy is not happy with an unused import

```suggestion
```